.github/workflows/tests.yml

```diff
@@ -22,6 +22,3 @@ jobs:
 
       - name: CGL
         run: Build/Scripts/runTests.sh -n -p ${{ env.php }} -s cgl -n
-
-      - name: Check Rst
```

Comment:
Why remove the check?

Reply:
Because we are checking the rst rendering with the guides in the new action documentation.yml now

Reply:
Ah sorry, yes.